### ISSUE TITLE
"Ñ" in variable?

### ISSUE THREAD

**Issue by aikiteresa** [kolber/stacey#153]
So I just installed stacey ... It looks good except "Ñame" on the index page has the "N" has the virgulilla on it. I tried editing the _shared.yml file ... but then the variable disappears altogether. Apologies if this is a basic question but I'm new to .yml. How do I fix the variable?

**Comment by pbatts** [comment]
Did you try replacing "Ñame" with a different value?

**Comment by aikiteresa** [comment]
I replaced "Ñame" with "Name" and the variable no longer displays.

**Comment by pbatts** [comment]
Replace
name: Ñame
with
name: Esme

**Comment by aikiteresa** [comment]
Again if I replace "name: Ñame" with "name: Esme" or "name: Name" or anything else ... the name variable no longer displays at all.

**Comment by pbatts** [comment]
Sorry, I don't know what to say.

I have tested it and it works fine for me.

Perhaps try a fresh install of Stacey.

Which template are you using?

**Comment by pbatts** [comment]
Did you get it working?

**Comment by aikiteresa** [comment]
No, I gave up. No idea why it's doing that.

**Comment by pbatts** [comment]
Which template are you using?

It drives me crazy not being able to figure this out and not being able to re-create the problem.

**Comment by aikiteresa** [comment]
Whatever the default template is when you download the zip from Github.

**Comment by pbatts** [comment]
I just downloaded a fresh .zip from https://github.com/kolber/stacey and it worked perfectly.

I would suggest downloading the .zip file again and give it a try with a fresh install. Only change "Ñame" in the `key: value` pair in the `_shared.yml` file.

**Comment by aikiteresa** [comment]
Tried that and still no go. The only thing I can think of is that the
encoding is getting weird when I edit. I'm just editing the _shared.yml
file in notepad. Saving it as All Files with the same name '_shared.yml'.

**Comment by pbatts** [comment]
Yes, that is very possible. I don't have a Windows machine, so, I am unable to test that. You may have to try a different editing program. There are several free ones specifically for web developers.

You can try changing the `_shared.yml` file extension to `_shared.txt`

Stacey will work with either `.yml` or `.txt` file extension.

**Comment by aikiteresa** [comment]
Super weird ... saving with the .txt extension gives the same result in
Notepad. Should have tried earlier, but editing it with GVIM works fine
(was avoiding it out of laziness ... don't use it enough to take advantage
of the commands). Guess Notepad does something strange ... maybe because
the end of line character is different from *.nix text editors? When I open
the _shared.yml file in Notepad, the line breaks aren't available so I get:

"name: Ñameprofession: Gräphic Designeremail: |  <hi@yourdomain.com>"
instead of the expected

"name: Ñame
profession: Gräphic Designer
email: |
  <hi@yourdomain.com>" which appears in GVIM

<shrug> FYI, I'm using Windows 8.1 64-bit.

Thanks for the support.

**Comment by pbatts** [comment]
Glad you got it figured out.

Best of luck with your project.
